### ISSUE TITLE
pyparsing.ParseException on step #1 of the tutorial

### ISSUE THREAD

**Issue by cogumbreiro** [habanero-rice/cnc-framework#5]
I am following the [CnC tutorial](https://wiki.rice.edu/confluence/download/attachments/17735877/CnC-OCR%20Tutorial.pdf). Running `ucnc_t` on the first example throws an exception.

Any idea what I'm doing wrong? Is the example deprecated?

I am referring to file: [0-blank.slate/Stencil1D.cnc](https://github.com/habanero-rice/cnc-framework/blob/master/examples/tutorial/0-blank.slate/Stencil1D.cnc)

```
$ ucnc_t  Stencil1D.cnc 
Traceback (most recent call last):
  File "/home/tiago/Work/cnc-framework/tools/unified_translator.py", line 299, in <module>
    UnifiedTranslator("ucnc_t").write_files()
  File "/home/tiago/Work/cnc-framework/tools/unified_translator.py", line 72, in __init__
    graphAst = parser.parseGraphFile(self.args.specfile)
  File "/home/tiago/Work/cnc-framework/tools/cncframework/parser.py", line 281, in parseGraphFile
    return parserErrorWrapper(cncGraphSpec, specPath)
  File "/home/tiago/Work/cnc-framework/tools/cncframework/parser.py", line 44, in parserErrorWrapper
    raise closestMatchError
pyparsing.ParseException: Expected delimiter: ';' (at char 317), (line:13, col:5)

```

**Comment by cogumbreiro** [comment]
Fixes issue #5.

https://github.com/habanero-rice/cnc-framework/issues/5

**Comment by DaoWen** [comment]
Those files were accidentally clobbered in an unrelated commit. I've reverted them. Sorry about that!
